Description: If monitoring is activated when closing audacity,
 the autosave file is not erased correctly
Bug-Ubuntu: https://launchpad.net/bugs/455990
Forwarded: yes
Author: David Henningsson <launchpad.web@epost.diwic.se>
diff -Nur -x '*.orig' -x '*~' audacity-1.3.9/src/Project.cpp audacity-1.3.9.new/src/Project.cpp
--- audacity-1.3.9/src/Project.cpp	2009-08-31 12:45:38.000000000 +0200
+++ audacity-1.3.9.new/src/Project.cpp	2009-10-20 04:31:41.221998664 +0200
@@ -1768,6 +1768,9 @@
       SetAudioIOToken(0);
       RedrawProject();
    }
+   else if (gAudioIO->IsMonitoring()) {
+      gAudioIO->StopStream();
+   }
 
    // These two lines test for an 'empty' project.
    // of course it could still have a history at this stage.
